src/components/inputs/input-text/input-text.tsx

```diff
@@ -14,6 +14,7 @@ export const InputText = React.forwardRef<RefNativeTextInput, InputTextProps>(
   (
     {
       isSearchInput = false,
+      isPhoneInput = false,
```

Comment:
Aren't isSearchInput and isPhoneInput mutually exclusive?

Shouldn't they be grouped in a property like below?

```tsx
inputType?: 'TEXT' | 'SEARCH' | 'PHONE'
```

Reply:
Good idea